react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-import chai, { expect } from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 import React from 'react';
-import {
-  Simulate,
-  renderIntoDocument,
-  scryRenderedDOMComponentsWithClass,
-  findRenderedDOMComponentWithClass
-} from 'react-dom/test-utils';
-import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
-import { findAllWithClass } from 'react-shallow-testutils';
-import Checkbox from './Checkbox';
-
-chai.use(sinonChai);
-
-const eventMatcher = sinon.match.instanceOf(SyntheticEvent);
+import { shallow } from 'enzyme';
```

Comment:
We were using render because we wanted to keep parity with previously written tests that had snapshots. Here's not need to use render as we don't want the full dom render